llvm/test/CodeGen/AMDGPU/add-xnull-regclass-bitwidth.mir

```diff
@@ -0,0 +1,12 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx90a -run-pass=early-machinelicm -run-pass=postmisched -o - %s | FileCheck %s
```

Comment:
Can do this in one run-pass with , separated pass names. but I'm not sure why you need to run 2 passes, or how either ne stresses this function